plugins/source/firestore/client/sync.go

```diff
@@ -93,14 +93,12 @@ func (c *Client) syncTable(ctx context.Context, table *schema.Table, res chan<-
 	return err
 }
 
-func (c *Client) syncTables(ctx context.Context, res chan<- *schema.Resource) error {
-	eg, ctx := errgroup.WithContext(ctx)
-	eg.SetLimit(len(c.Tables))
```

Comment:
No point of setting the limit to the number of tables as that's exactly the number of Go Routines that are created in the loop below